src/components/contactsList/ContactsList.jsx

```diff
@@ -3,13 +3,13 @@
 import s from './ContactsList.module.css'
 
 export default function ContactsList({ contacts, setChatId }) {
-  const list = contacts.map((name) => (
+  const list = contacts.map((item) => (
```

Comment:
лучше семантичные имена юзать:
```suggestion
  const list = contacts.map((contact) => (
```